src/Sulu/Bundle/SnippetBundle/Tests/Unit/Snippet/DefaultSnippetManagerTest.php

```diff
@@ -63,10 +63,10 @@ public function saveDataProvider()
             ['sulu_io', 'de', 'test', 'test_homepage', '123-123-123', false],
             ['sulu_io', 'de', 'test', 'test_homepage', '123-123-123', true, false],
             ['sulu_io', 'de', 'test', 'test_homepage', '123-123-123', false, false],
-            ['sulu_io', 'de', 'test', 'testCase', '123-123-123'],
-            ['sulu_io', 'de', 'test', 'testCase', '123-123-123', false],
-            ['sulu_io', 'de', 'test', 'testCase', '123-123-123', true, false],
-            ['sulu_io', 'de', 'test', 'testCase', '123-123-123', false, false],
+            ['sulu_io', 'de', 'test', 'testcase', '123-123-123'],
+            ['sulu_io', 'de', 'test', 'testcase', '123-123-123', false],
+            ['sulu_io', 'de', 'test', 'testcase', '123-123-123', true, false],
+            ['sulu_io', 'de', 'test', 'testcase', '123-123-123', false, false],
```

Comment:
@alexander-schranz You've been looking into that issue with the parameter bag once, right? Is this the right way to fix it?

Reply:
In the constructor of the [DefaultSnippetManager]( https://github.com/sulu/sulu/blob/1.6.8/src/Sulu/Bundle/SnippetBundle/Snippet/DefaultSnippetManager.php#L62) the `array_change_key_case` need to be removed the test case need to have `testCase` as before.